lib/core/device.js

```diff
@@ -217,7 +217,11 @@ device.detectUsb = function(driverId, cb) {
     vendorId: driverManifest.vendorId,
     productId: driverManifest.productId
   };
-  chrome.usb.getDevices(identification, function(results) {
```

Comment:
So from what I understand of your explanation from elsewhere, there are disadvantages to doing the `chrome.serial.getDevices` when multiple devices might be connected. If needing to use it is a limitation that only affects Windows users so far, could we keep `chrome.usb.getDevices` for Mac users? Maybe we don't make that change today since we want to get this out ASAP, but what do you think?